src/styles/filterandsort.css

```diff
@@ -0,0 +1,24 @@
+.filter__and__sort{
+    display: none;
+}
+@media screen and (min-width : 769px) {
+.filter__and__sort{
+    display: flex;
+    flex-direction: column;
+    margin-left: 2rem;
+    background-color: var(--white);
+    height:fit-content;
+    position: sticky;
+    top: 90.2px;
```

Comment:
add some bgcolor to this button , to add some visibility

Reply:
I tried it nor going with design